src/main/java/org/jenkinsci/plugins/saml/OpenSAMLWrapper.java

```diff
@@ -103,8 +103,9 @@ protected SAML2Client createSAML2Client() {
         if (samlPluginConfig.getEncryptionData() != null) {
             config.setWantsAssertionsSigned(true);
             config.setKeystorePath(samlPluginConfig.getEncryptionData().getKeystorePath());
-            config.setKeystorePassword(samlPluginConfig.getEncryptionData().getKeystorePassword());
-            config.setPrivateKeyPassword(samlPluginConfig.getEncryptionData().getPrivateKeyPassword());
+            // TODO does this really accept blanks? its constructor seems to imply it does not
```

Comment:
it accept blank values but you can not have a keystore with blank password, so it will fail but for external reasons